tests/test_json.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+from flask import Flask, jsonify
+
+from flask_jwt_extended import JWTManager, jwt_required, jwt_refresh_token_required, create_access_token, create_refresh_token
+from tests.utils import get_jwt_manager
+
+
+@pytest.fixture(scope='function')
+def app():
+    app = Flask(__name__)
+    app.config['JWT_SECRET_KEY'] = 'foobarbaz'
+    app.config['JWT_TOKEN_LOCATION'] = 'json'
+    JWTManager(app)
+
+    @app.route('/protected', methods=['POST'])
+    @jwt_required
+    def access_protected():
+        return jsonify(foo='bar')
+
+    @app.route('/refresh', methods=['POST'])
+    @jwt_refresh_token_required
+    def refresh_protected():
+        return jsonify(foo='bar')
+
+    return app
+
```

Comment:
Could we add one more unit test to this file, this working with the default json keys?